### ISSUE TITLE
Does the project support train batch norm in multiple gpus?

### ISSUE THREAD

**Issue by John1231983** [NifTK/NiftyNet#45]
I have 4 gpus and I want to train my network that includes batch norm layer on the machine. As we know, the segmention performance will increase if we consider a larger batch size and train the batch norm
 However, the tensorflow did not support to syn. batch norm on multiple gpus. Does this project support it? Thanks

**Comment by wyli** [comment]
Hi @John1231983 currently the moving average of statistics from one of the GPUs will be used at inference time, and each GPU computes statistics independently during training.
